website/docs/plugin/framework/migrating/attributes-blocks/blocks.mdx

```diff
@@ -14,6 +14,38 @@ This page explains how to migrate nested blocks that are not computed (i.e., do
+| DiffSuppressFunc      | [PlanModifiers](/framework/migrating/resources/plan-modification#framework) field on attribute within block or implementation of [ResourceWithModifyPlan](/plugin/framework/migrating/resources/plan-modification#framework) interface                                                                                                        |
+| DiffSuppressOnRefresh | [Read](/plugin/framework/migrating/resources/crud) method on resource                                                                                                                                                                                                                                                                         |
+| Default               | [PlanModifiers](/framework/migrating/resources/plan-modification#framework) field on attribute within block or implementation of [ResourceWithModifyPlan](/plugin/framework/migrating/resources/plan-modification#framework) interface                                                                                                        |
+| DefaultFunc           | [PlanModifiers](/framework/migrating/resources/plan-modification#framework) field on attribute within block or implementation of [ResourceWithModifyPlan](/plugin/framework/migrating/resources/plan-modification#framework) interface                                                                                                        |
+| Description           | `Description` field on attribute                                                                                                                                                                                                                                                                                                              |
+| InputDefault          | N/A - no longer valid                                                                                                                                                                                                                                                                                                                         |
+| StateFunc             | Requires implementation of bespoke logic before storing state, for instance in resource [Create method](plugin/framework/migrating/resources/crud#framework-1)                                                                                                                                                                                |
+| Elem                  | N/A - define `Type` as [ListNestedBlock](/plugin/framework/migrating/attributes-blocks/blocks) or [SetNestedBlock](/plugin/framework/migrating/attributes-blocks/blocks)                                                                                                                                                                      |
```

Comment:
I wonder if it might be good to phrase it this way?

```suggestion
| Elem                  | `NestedObject` within block                                                                                                                                                                      |
```